src/sp/it/pl/main/App.kt

```diff
@@ -562,5 +562,12 @@ class App: Application(), Configurable<Any> {
         onNewInstanceHandlers += Consumer { parameterProcessor.process(it) }
     }
 
-    private fun File.initForApp() = apply { isValidatedDirectory(this) }
+    private fun File.initForApp() = apply {
```

Comment:
I'd rather not call this init, since it doesn't initialize anything. Maybe validate or check?

Reply:
So basically Id like the validation checks to remain an implementation detail. This method is an initialization logic an app needs to invoke for the file so it (the app) initializes properly.